<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormViewTable</class>
 <widget class="QWidget" name="FormViewTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Результаты проверки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Тип теплосчётчика</string>
     </property>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Годность</string>
     </property>
     <widget class="QComboBox" name="comboBox_usable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButton_filtersReset">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Сбросить фильтры</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="10">
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>750</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QToolButton" name="toolButton_updateTable">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Обновить таблицу</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Пользователь</string>
     </property>
     <widget class="QComboBox" name="comboBox_user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkBox_allNotes">
     <property name="text">
      <string>Показывать все записи</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>118</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>Дата</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Начало диапазоа</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_start">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Конец диапазона</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_stop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_date">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
